src/foam/nanos/dig/DigWebAgent.java

```diff
@@ -115,6 +117,12 @@ public void execute(X x) {
       logger.debug("predicate", pred.getClass(), pred.toString());
       dao = dao.where(pred);
 
+      SinkParser sinkParser = new SinkParser(cInfo);
+
+      StringPStream ps = new StringPStream(fields);
+      ParserContextImpl x_ = new ParserContextImpl();
+      ps = (StringPStream) sinkParser.parse(ps, x_);
```

Comment:
Below this line, print out ps.value to see if the SinkParser parsed it as you expected.